app/main.py

```diff
@@ -1,3 +1,65 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, kilometers: int) -> None:
+        self.km = kilometers
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(
+            self, new_distance: Union["Distance", int, float]
+    ) -> "Distance":
+        if isinstance(new_distance, (int, float)):
+            return Distance(self.km + new_distance)
+        else:
```

Comment:
`else` operator is redundant here. Just `return Distance(self.km + new_distance.km)`
Fix in places below as well